src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,19 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
+
+    List<Car> getAll();
+
+    Car update(Car car);
+
+    boolean delete(Long id);
+
+    List<Car> getAllByDriver(Long driverId); // pid pytannyam...
```

Comment:
PR is not the correct place for questions. We have Slack for it)
```suggestion
    List<Car> getAllByDriver(Long driverId);
```

Reply:
My bad!) It was only for me, forgot to delete)